projects/libs/core-library-typescript/src/common-library/startup-status.ts

```diff
@@ -0,0 +1,14 @@
+export interface StartupStatus {
+  name: string;
+  status: STARTUP_STATUS;
+  childStartupStatuses?: StartupStatus[];
```

Comment:
Please replace with `services`.

Reply:
This might not always relate just to services. Had a discussion with Shai about it.

Reply:
Shai does not define the requirements for this task. His input is valuable but I specifically asked for a certain API which we discussed in the beginning of last week.

Reply:
You mean replace childStartupStatuses with services?